Building DAG of jobs...
Full Traceback (most recent call last):
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/__init__.py", line 544, in snakemake
    export_cwl=export_cwl)
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/workflow.py", line 421, in execute
    dag.init()
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 122, in init
    job = self.update([job], progress=progress)
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 628, in update
    raise exceptions[0]
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 600, in update
    progress=progress)
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 670, in update_
    raise ex
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 663, in update_
    progress=progress)
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 628, in update
    raise exceptions[0]
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 600, in update
    progress=progress)
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 670, in update_
    raise ex
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 663, in update_
    progress=progress)
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 628, in update
    raise exceptions[0]
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 600, in update
    progress=progress)
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 670, in update_
    raise ex
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 663, in update_
    progress=progress)
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 628, in update
    raise exceptions[0]
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 600, in update
    progress=progress)
  File "/Users/morril01/anaconda3/envs/snakemake-globalDA/lib/python3.6/site-packages/snakemake/dag.py", line 679, in update_
    raise MissingInputException(job.rule, missing_input)
snakemake.exceptions.MissingInputException: Missing input files for rule simulate_for_comparison_multiple:
../data/assessing_models_simulation/GenerationInoREscaledbeta

MissingInputException in line 192 of /Users/morril01/Documents/PhD/GlobalDA/code/Snakefile:
Missing input files for rule simulate_for_comparison_multiple:
../data/assessing_models_simulation/GenerationInoREscaledbeta
